catalog.html

```diff
@@ -131,7 +131,22 @@ <h2 class="catalog-heading">All items</h2>
                     <input type="search" class="search-nft" placeholder="Search NFT">
 
                     <button type="submit" class="search-button">
-                        <img src="./img/loupe-icon.svg" alt="Loupe icon" class="search-img">
+                        <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+                            <mask id="mask0_1_8344" style="mask-type:luminance" maskUnits="userSpaceOnUse" x="1" y="1" width="18" height="17">
```

Comment:
We have quality criteria 
> ❤️ 3.2. Extract images without style filters from the design.

Could you please try to export image again from the design without any filters? And it would be better to add it into the sprite

Reply:
I see you moved this svg to the sprite. It's good, but you still have `style="mask-type:luminance"`. I think, you can just remove this CSS. As I understand, it won't change anything.